### PR TITLE
Make dev/ci image derive from nvidia/cuda

### LINKED ISSUE #480: Make our GPU enabled image derive from nvidia/cuda instead of extending ci/dev base image
c.f. #473 and issues with CUDA versions from 9.2 onwards
c.f. #451 to install Trilinos with CUDA enabled

### PR DESCRIPTION
**Pull request by dalg24**
Rather than installing the CUDA toolchain on top of our base image

Closes #480

### CODE REVIEW

**Comment by dalg24** [comment]
Dammit Andrey.  Will rollback the update of Trilinos

**Comment by dalg24** [comment]
PS this is ready for review.  I will **not** rebase on #528 and build new images

File: docker/nvidia/setup_nvidia_docker_compose.py [hunk 1]
```diff
@@ -120,19 +120,7 @@ def get(self, endpoint):
...
     config['services'][service]['volumes'] = volumes
     config['services'][service]['environment'] = ['GPU_ARCH=' + gpu_arch]
-    config['services'][service]['image'] = service + '_cuda_toolchain'
-    dockerfile = 'Dockerfile_' + service + '_cuda_toochain_generated'
-    config['services'][service]['build'] = {
-        'context': '.',
-        'dockerfile': dockerfile,
-        'args': [
-            'CUDA_VERSION=' + cuda_version,
-            'CUDA_PKG_VERSION=' + cuda_pkg_version,
```
Comment (Rombur) [inline docker/nvidia/setup_nvidia_docker_compose.py]: I think that `cuda_pkg_version` is not used anymore. So we can remove all the code associated with it.

**Comment by dalg24** [comment]
> You can tag the new images and then merge.

I have already tagged them as latest and pushed to Docker Hub.  This is ready to merge.

File: docker/nvidia/setup_nvidia_docker_compose.py (hunk 1 above)
Comment (dalg24) [reply docker/nvidia/setup_nvidia_docker_compose.py]: You are correct.  I will address that in another PR when we move towards nvidia-docker 2.0 if that's OK with you.

**Pull request merged**